src/main/java/jeryl/fyp/model/ModelManager.java

```diff
@@ -111,6 +112,13 @@ public void setStudent(Student target, Student editedStudent) {
         fypManager.setStudent(target, editedStudent);
     }
 
+    @Override
+    public Student getStudent(StudentId studentId) {
```

Comment:
Do you think it's better to have the naming to be `getStudentByStudentId` instead?

Reply:
Yeap seems more appropriate I'll change it then!